src/status_im2/contexts/wallet/events.cljs

```diff
@@ -102,17 +103,19 @@
                                       :event  :wallet/get-wallet-token
                                       :params addresses})}]]]})))
 
-(defn- fix-chain-id-keys
+(defn- fix-balances-per-chain
   [token]
-  (update token :balances-per-chain update-keys (comp utils.number/parse-int name)))
+  (-> token
+      (update :balances-per-chain update-vals #(update % :raw-balance money/bignumber))
+      (update :balances-per-chain update-keys (comp utils.number/parse-int name))))
 
 (rf/reg-event-fx
  :wallet/store-wallet-token
  (fn [{:keys [db]} [raw-tokens-data]]
    (let [tokens     (-> raw-tokens-data
                         (update-keys name)
                         (update-vals #(cske/transform-keys csk/->kebab-case %))
-                        (update-vals #(mapv fix-chain-id-keys %)))
+                        (update-vals #(mapv fix-balances-per-chain %)))
```

Comment:
Here, when the token raw-balance is received, is being stored as  bignumber instances